### ISSUE TITLE
Fetch manifest from <link /> tag

### ISSUE THREAD

**Issue by toniengelhardt** [notangelmario/paquet#83]
**Is your feature request related to a problem? Please describe.**

I'm using the [Nuxt PWA module](https://pwa.nuxtjs.org/) for my PWA and unfortunately it generates dynamic file names like `manifest.4803dada.json` to avoid caching, which change at every deployment.

You can have a look here: [https://pwa.journalisticapp.com](https://pwa.journalisticapp.com)

**Describe the solution you'd like**

It would be nice if paquet would fetch the manifest from the `<link rel="manifest" />` head tag of the page instead of a hardcoded URL. That would also make the submission a bit smoother.

**Additional context**

--

**Comment by notangelmario** [comment]
Hi, @toniengelhardt! Yes, I have been thinking about this for quite some time, but never got around to do it.

This issue will be addressed after #82 closes.

**Comment by notangelmario** [comment]
@toniengelhardt thanks! I have always been a fan of the PWA concept. Even with it's cons, PWAs should be given more attention because they can also have their place in the user's device.

**Comment by toniengelhardt** [comment]
> @toniengelhardt thanks! I have always been a fan of the PWA concept. Even with it's cons, PWAs should be given more attention because they can also have their place in the user's device.
> 
> Edit: Also, would you like to add [pwa.journalisticapp.com](https://pwa.journalisticapp.com/) to Paquet once I merge the pull request? I don't want to put you through the hassle of creating another issue.

Yes, definitely! I was in the process of submitting it when I encountered the issue with the manifest.

I also have a few more:

- [RepoTracker](https://repo-tracker.com) (GitHub repository stats and insights)
- [MΞRKLIN](https://merklin.xyz) (Open-source EVM wallet explorer made with Nuxt)
- [WebAPI check](https://webapicheck.com) (Test WebAPI compatibility of any device)

**Comment by notangelmario** [comment]
Here are your apps:
https://paquet.shop/app/8654fbf9-181d-4b22-9b30-d83db968e790
https://paquet.shop/app/2acc9ccb-b10e-4ec8-8a0a-b1918cedb525
https://paquet.shop/app/213305c3-3a07-4805-8c1e-9182af32d5b6
https://paquet.shop/app/29a9e456-cefa-460b-bf08-c0a24737b1c5

First of all, I have to say, I fell in love with Journalistic. I will certainly take some notes when creating the about page of Paquet.

**Comment by notangelmario** [comment]
@all-contributors please add @toniengelhardt for the idea
